provider/pkg/provider/provider.go

```diff
@@ -763,25 +781,20 @@ func (k *kubeProvider) Configure(_ context.Context, req *pulumirpc.ConfigureRequ
 			}
 			warningConfig := rest.CopyConfig(config)
```

Comment:
Do we need to deepcopy the config object if we're just assigning it back to config? Could we simplify to just:
```
config.WarningHandler = rest.NoWarnings{}
```